### ISSUE TITLE
I was wondering, may I add my journal too?

### ISSUE THREAD

**Issue by reihanian** [stop-predatory-journals/stop-predatory-journals.github.io#48]
Hi,

I was wondering, may I add my journal too? the information came from three different authors who is expert in their area.

https://www.myhealthyexperience.com

**Comment by gobenavides** [comment]
I don't think this is academic journal, but only a questionable fitness/health page

**Comment by reihanian** [comment]
> I don't think this is academic journal, but only a questionable fitness/health page

So what would be definition of a journal? I am not talking about Americal Journal of Medical Science, but isn't find the common issues people are dealing with everyday and after using a proper method of logic then provide a solution which will hold for N and N+1?
It is clear that Journals that are paying attention to moer important issues have more impact factor.

Thanks,
Iman
